Libplanet/Tx/InvalidTxException.cs

```diff
@@ -15,13 +15,15 @@ public abstract class InvalidTxException : Exception
         /// <see cref="Transaction{T}.Id"/>.  It is automatically included to
         /// the <see cref="Exception.Message"/> string.</param>
         /// <param name="message">A descriptive error message for programmers.
+        /// </param> <param name="inner">A Inner-exception.
         /// </param>
-        protected InvalidTxException(TxId txid, string message)
-            : base($"{txid.ToHex()}: {message}")
+#nullable enable
+        protected InvalidTxException(TxId txid, string message, Exception? inner)
+            : base($"{txid.ToHex()}: {message}", inner)
         {
             TxId = txid;
         }
```

Comment:
Two things:
- It'd be better to have `InvalidTxExceptioin(TxId, string)` *and* `InvalidTxException(TxId, string, Exception)` separately.
  - This would get rid of unnecessary cluttered use of `null`s everywhere.
  - This would be in better accordance to built-in `Exception`s.
- Change the parameter name `inner` to `innerException`. Again, this is what built-in `Exception`s use. 😗

Reply:
Updated!